activesupport/lib/active_support/evented_file_update_checker.rb

```diff
@@ -21,7 +21,13 @@ def initialize(files, dirs = {}, &block)
         # Loading listen triggers warnings. These are originated by a legit
         # usage of attr_* macros for private attributes, but adds a lot of noise
         # to our test suite. Thus, we lazy load it and disable warnings locally.
-        silence_warnings { require 'listen' }
+        silence_warnings do
+          begin
+            require 'listen'
+          rescue LoadError => e
+            raise LoadError, "Could not load the 'listen' gem. Add `gem 'listen'` to the development section of your Gemfile", e.backtrace
```

Comment:
I would replace "development section" with "development group" here as that is what its called in Gemfile.